spec/system/curriculum_content/user_view_curriculum_content_details_spec.rb

```diff
@@ -74,4 +74,18 @@
 
     expect(current_path).to eq schedule_item_path(schedule_item.id)
   end
+
+  it 'and curriculum content doesnt exists' do
+    user = create(:user)
+    create(:profile, user: user)
+    schedule_item = build(:schedule_item, id: 99, title: 'TDD com Rails', description: 'Introdução a programação com TDD')
```

Comment:
Objeto schedule_item não possui id, alterado para code conforme proposta de events-app
```suggestion
    schedule_item = build(:schedule_item, id: 99, title: 'TDD com Rails', description: 'Introdução a programação com TDD')
```